tests/test_arrow.py

```diff
@@ -1754,6 +1754,11 @@ def test_span_week(self):
         assert floor == datetime(2013, 2, 11, tzinfo=tz.tzutc())
         assert ceil == datetime(2013, 2, 17, 23, 59, 59, 999999, tzinfo=tz.tzutc())
 
+        floor, ceil = self.arrow.span("week", week_start=6)
+
+        assert floor == datetime(2013, 2, 9, tzinfo=tz.tzutc())
+        assert ceil == datetime(2013, 2, 15, 23, 59, 59, 999999, tzinfo=tz.tzutc())
+
```

Comment:
We still need a test that checks what happens when a bad week_start number is passed.

Reply:
You're right. There is no check whether the parameter week_day is between 1 and 7. Setting week_day to 55 would return a week in the future. I could raise a `ValueError` if `1 <= week_day <= 7` is not met.

Reply:
@Alee008 sounds like a good idea along with a test for that.